cdk/tsconfig.json

```diff
@@ -1,31 +1,31 @@
 {
+	"extends": "@guardian/tsconfig",
 	"ts-node": {
 		"compilerOptions": {
 			"module": "CommonJS"
 		}
 	},
 	"compilerOptions": {
 		"target": "ES2020",
-		"module": "ES2020",
-		"moduleResolution": "node",
 		"lib": ["ES2020"],
 		"declaration": true,
-		"strict": true,
 		"noImplicitAny": true,
 		"strictNullChecks": true,
-		"esModuleInterop": true,
 		"noImplicitThis": true,
 		"alwaysStrict": true,
-		"noUnusedLocals": false,
 		"noUnusedParameters": false,
-		"noImplicitReturns": true,
 		"noFallthroughCasesInSwitch": false,
 		"inlineSourceMap": true,
 		"inlineSources": true,
 		"experimentalDecorators": true,
 		"strictPropertyInitialization": false,
 		"typeRoots": ["./node_modules/@types"],
-		"outDir": "dist"
+		"outDir": "dist",
+		"allowJs": true,
+		"isolatedModules": true,
+		"module": "ES6",
```

Comment:
do you need this value specifically? the default from the config is `esnext`

Reply:
I've removed the module from this tsconfig.json file. However, I had to add the "module": "commonjs" to monitoring/tsconfig.json